kie-wb-webapp-common/src/main/webapp/login.jsp

```diff
@@ -33,23 +35,23 @@
   <img id="logo" src="images/kie-ide.png" alt="KIE IDE Logo" title="Powered By Drools/jBPM"/>
 
   <div id="login-wrapper" class="png_bg">
-    <div id="login-top">    
+    <div id="login-top">
     </div>
 
     <div id="login-content" class="png_bg">      
       <form action="j_security_check?locale=<%=locale%>" method="POST">
         <fieldset>
           <c:if test="${param.message != null}">
-            <h3><i18n:message key="loginFailed">Login failed: Not Authorized</i18n:message></h3>
+            <h3><i18n:message key="loginFailed"></i18n:message></h3><br/>
```

Comment:
I'm beginning to think you've just removed the default text as it's substituted by the taglib at compile time?